### ISSUE TITLE
GearSwap: runtime error on line 228

### ISSUE THREAD

**Issue by Sechs** [Windower/Lua#863]
Just logged and getting this error I never got before:

GearSwap: Lua error (runtime) - .../addons/GearSwap/GearSwap.lua:228: attempt to index field 'vitals' (a nil value)

Just noticed this happens also on line 294 while zoning. I get two lines

> inventory 0
> GearSwap: Lua error (runtime) - .../addons/GearSwap/GearSwap.lua:294: attempt to index field '?0 (a nil value)

**Comment by Sechs** [comment]
Ehr nevermind, just saw Byrth write that these issues are known and will be fixed in a push later today.
Sorry!

**Comment by Byrth** [comment]
These should be fixed now. Re-run launcher to get the most recent version. Please let me know if they still happen.

**Comment by Sechs** [comment]
Not getting 294 and 228, but getting 236 now. On a different lua file though

...../addons/GearSwap/helper_functions.lua:236 attempt to index field '?' (a nil value)

**Comment by Sechs** [comment]
Got another error twice in a row on gearswap.lua line 407, same nil index attempt thing.
Don't know what was happening when I got those lines, I think I was trying to change an instrument from Horn to Harp or vice-versa.

Or rather, equipping two "empty" set, which only has harp or horn as items inside.
Also, I was engaged at the time.

Edit:
totally happens when I engage/disengage, a bit randomly.

Edit2:
Also 294 is back. Got it when I zoned OUT form Hazhalm Training Grounds.
Strangely enough it didn't happen in Adoulin>Caedarva and Caedarva>Hazhalm.
It did happen zoning Caedarva>Nashmau.
So it's a bit random too I guess?
